backend/repositories/match-repository.js

```diff
@@ -3,6 +3,7 @@ const ratingCalculator = require("../rating/rating-calculator")
 const { InputError } = require('../errors/input-error')
 const { NotFoundError } = require('../errors/not-found-error')
 const bot = require('../bot');
+const ADD_MATCH_COOLDOWN = 60
```

Comment:
On a second thought, is 60 second enough? 🤔
There can be a scenario when user tries to add a match while the server on heroku is sleeping. The request will still be sent and heroku will wait for the server to wake up to process it, but the user in the meanwhile can refresh the page, go grab a coffee yada yada, and when they come back the minute could have already elapsed and they add a duplicate match since they won't see that the first request actually came through.
I would go with 5 minutes just to be safe.

Reply:
Ok, I'll make it configurable.